aeolic.go

```diff
@@ -45,6 +48,19 @@ func NewWithMap(apiKey string, templateMap map[string]string) Client {
 	}
 }
 
+func NewWithFS(apiKey string, dir fs.FS, templateDir string) (Client, error) {
```

Comment:
NewWithFS is unused?

Reply:
The others New funcs have examples in cmd, I can add one if needed